ethcore/client-traits/src/io_message.rs

```diff
@@ -16,13 +16,12 @@
 
```

Comment:
it is a bit odd to have these types in `client-traits`, is there a better location for it?

Reply:
You are not wrong. :)

I could move it to `common-types`. Would that be better?